test/nodes/test_reader.py

```diff
@@ -1,4 +1,7 @@
 import math
+import os
+import tempfile
```

Comment:
I think this import is not used.

Reply:
Aha, that's because we don't check imports for tests, will fix - thanks @bogdankostic